omod-common/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/MainResourceControllerTest.java

```diff
@@ -133,6 +133,29 @@ public MockHttpServletRequest newPostRequest(String requestURI, String content)
 		return request;
 	}
 	
+	public MockHttpServletRequest newPutRequest(String requestURI, Object content) {
+		MockHttpServletRequest request = request(RequestMethod.PUT, requestURI);
+		try {
+			String json = new ObjectMapper().writeValueAsString(content);
+			request.setContent(json.getBytes("UTF-8"));

```

Comment:
Instead of the line above, why don't you just call newPutRequest(requestURI, json)?